server/services/gameLogic.js

```diff
@@ -185,4 +185,4 @@ const playerHasVoted = (username) => votes.filter(vote => vote.username === user
 const allPlayersPlayedCard = () => players.every(player => playerHasPlayedCard(player.username));
 
 /* Returns true if all players (apart from the current player) have voted this round */
-const allPlayersVoted = () => players.filter(player => player.username !== currentPlayer.username).every(player => playerHasVoted(player.username));
+const allPlayersVoted = () => players.every(player => player.username === currentPlayer.username || playerHasVoted(player.username));
```

Comment:
I think the way it was before was more readable, how come you changed it?

Reply:
Ronan said there's no need of looping through the list twice

Reply:
I could change it back to what it was before - pick readability or efficiency? :D I don't mind in this situation

Reply:
Yeah, that's the choice. I just wanted to highlight another option.